deploy/charts/harvester-journal/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
This looks like a Harvester-specific chart. The convention is to create the chart in the https://github.com/harvester/charts repo and publish it. Then we can use it as a dependency chart in the `harvester/harvester` repo. You can check [other charts](https://github.com/harvester/charts/tree/master/charts) as an example. 

cc @guangbochen

Reply:
create PR for harvester-charts: https://github.com/harvester/charts/pull/94/